pkg/analyser/analyzer.go

```diff
@@ -49,3 +67,18 @@ func findCorrespondingRes(resources []resource.Resource, res resource.Resource)
 	}
 	return -1, nil, false
 }
+
+func getField(t reflect.Type, path []string) (reflect.StructField, bool) {
```

Comment:
Scope it to analyzer struct 

`func (a *Analyzer) getField(t reflect.Type, path []string) (reflect.StructField, bool)`

It does not makes sense to expose this method in the whole package

Reply:
Done 7f5211c